src/main/java/Util.java

```diff
@@ -0,0 +1,39 @@
+import javafx.scene.image.Image;
+import javafx.scene.paint.Color;
+
+import java.io.FileInputStream;
+import java.net.URL;
+
+public class Util {
+
+    static int width = 750;
+    static int height = 600;
+    static int spaceBtwRows = 15;
+    static int sidePadding = 10;
+    static int drawingBoxLeftPadding = 410;
+    // these values are user as color types
+    static String user = "user";
+    static String computer = "computer";
+    static String intercept = "intercept";
+    static String kenoImageURL = "https://lh3.googleusercontent.com/yw7Y7vidKLuVvYGEolEcheo36KnJeV4zksjxXD6mSueZJOeS8RdCOu6EwuJqVrtdIwI";
```

Comment:
What does this URL do?

Reply:
I had it there earlier because I was trying my hands on using an image as background. But it didn't really work. You can get rid of that.